### PR TITLE
add hoverable icon to toggle password visibilty on woocommerce forms

### PR DESCRIPTION
**Pull request by BrandynL**
### All Submissions:

* [x ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Added a javascript only solution to display a password visibilty icon on all woocommerce login form password inputs, as I found a related open issue (#24823 ) and i thought it was a good idea for my first pull request!

### How to test the changes in this Pull Request:

1. Visit any woocommerce login form and hover over the new "eye" visibility icon on the password input

### Other information:

* [x ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x ] Have you successfully run tests with your changes locally?

### Changelog entry

>Added hover icon on password input fields to briefly toggle input text visibility for browsers with javascript enabled

### CODE REVIEW

File: assets/js/frontend/woocommerce.js [hunk 1]
```diff
@@ -78,4 +78,17 @@ jQuery( function( $ ) {
 			}, 1000 );
 		}
 	};
+
+	// Show password visiblity hover icon on woocommerce forms
+	$('.woocommerce form #password').wrap('<span class="password-input"></span>');
```
Comment (claudiosanches) [inline assets/js/frontend/woocommerce.js]: This code isn't following the WordPress coding standards, see https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Thanks for your help, but there's a few things to fix before we can merge this PR.

File: assets/js/frontend/woocommerce.js (hunk 1 above)
Comment (BrandynL) [reply assets/js/frontend/woocommerce.js]: Woops - first timer here :)

I reviewed those standards and I believe I'm within spec now.

**Review by claudiosanches — CHANGES_REQUESTED** [review]
Seems like you compressed the files using something else than our Gruntfile, that's why is including changes to several files that shouldn't be included into this PR.
Please remove those files and squash those commits into one.

**Pull request closed without merging**